python/llm/test/inference/test_transformers_api.py

```diff
@@ -17,11 +17,11 @@
 
 import unittest
 import os
-
+import pytest
 import time
 import torch
-from bigdl.llm.transformers import AutoModel
-from transformers import AutoTokenizer
+from bigdl.llm.transformers import AutoModel, AutoModelForCausalLM, AutoModelForSpeechSeq2Seq
+from transformers import AutoTokenizer, AutoModel as TAutoModel, AutoModelForCausalLM as TAutoModelForCausalLM
```

Comment:
Seems that this `AutoModel as TAutoModel, AutoModelForCausalLM as TAutoModelForCausalLM` is not needed to import